vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -107,7 +107,7 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.28.5", # for k8s 1.28.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.29.0", # for k8s 1.29.x
     # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-    "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.15",
+    "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.17",
```

Comment:
Please keep the old image if the disk is not full. Otherwise the customers  who are using v3.1.15 will miss the cache.

Reply:
change to keep 3.1.15 as well